package.json

```diff
@@ -1,17 +1,11 @@
 {
   "name": "fxa-jwtool",
-  "version": "0.6.0",
   "description": "a JWT utility",
-  "main": "index.js",
-  "scripts": {
-    "test": "tap test"
+  "version": "0.6.0",
+  "author": "Mozilla (https://mozilla.org/)",
```

Comment:
Sorry, I think I just copied this from fxa-auth-server. Feel free to change the author to your name since you're the sole author of this stuff.